tests/scripts/thread-cert/Cert_5_6_05_NetworkDataRegisterAfterAttachRouter.py

```diff
@@ -109,7 +109,7 @@ def test(self):
             if addr[0:3] == '200':
                 self.assertTrue(self.nodes[LEADER].ping(addr))
 
-        self.nodes[ROUTER].add_prefix('2001:2:0:3::/64', 'pacs')
+        self.nodes[ROUTER].add_prefix('2001:2:0:3::/64', 'paos')
```

Comment:
Add some test to verify that OT handles `off-mesh` prefixes properly?

Reply:
Added a test in a new commit.